modules/features/org.apache.synapse.samples.feature/pom.xml

```diff
@@ -0,0 +1,80 @@
+<!--
+ ~ Copyright (c) 2005-2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~ WSO2 Inc. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied.  See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+     <parent>
+       <artifactId>synapse-features</artifactId>
+       <version>2.1.7-wso2v118-SNAPSHOT</version>
+       <groupId>org.apache.synapse</groupId>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.apache.synapse.samples.feature</artifactId>
+    <packaging>pom</packaging>
+    <name>WSO2 Carbon - Synapse Samples Feature</name>
+    <url>http://wso2.org</url>
+    <description>This feature contains the synapse samples</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.synapse</groupId>
+            <artifactId>synapse-samples</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.synapse</groupId>
+            <artifactId>org.apache.synapse.wso2.feature</artifactId>
+            <version>2.1.7-wso2v96-SNAPSHOT</version>
```

Comment:
lets take this from project version

Reply:
Done in https://github.com/wso2/wso2-synapse/pull/1391/commits/d9ab8a682e834be7903af1bfd95939f6014c9597